src/main/java/com/pretz/parkingmanager/domain/ParkingSession.java

```diff
@@ -0,0 +1,45 @@
+package com.pretz.parkingmanager.domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Entity
+@Table(name = "PARKING_SESSION")
+public final class ParkingSession {
```

Comment:
czy to naprawdę jest domena? - ja tu widzę więcej JPA niż domeny

Reply:
Chodzi o to, że nie ma żadnych metod poza getterami i setterami i wychodzi anemiczna domena?